react/features/base/lib-jitsi-meet/actions.js

```diff
@@ -42,11 +42,25 @@ export function initLib() {
     return (dispatch: Dispatch<any>, getState: Function): void => {
         const state = getState();
         const config = state['features/base/config'];
+        const { enableHdAudio } = config;
 
         if (!config) {
             throw new Error('Cannot init lib-jitsi-meet without config');
         }
 
+        if (enableHdAudio) {
+            Object.assign(config, {
+                ...config,
+                channelCount: 2,
+                disableAP: true,
+                enableNoAudioDetection: false,
+                enableNoisyMicDetection: false,
+                enableTalkWhileMuted: false,
+                opusMaxAverageBitrate: 510000,
```

Comment:
This is the maximum supported value, can be tweaked if we want.

Reply:
IMHO that value is ridiculously high, but yeah, we can test.

Reply:
We souldn't be doing this here, but in LJM.